apps/ios/ru.lproj/Localizable.strings

```diff
@@ -388,6 +391,9 @@
 /* No comment provided by engineer. */
 "Unexpected error: %@" = "Неожиданная ошибка: %@";
 
+/* No comment provided by engineer. */
+"Unless your contact deleted the connection or this link was already used, it might be a bug - please report it. To connect, please ask your contact to create another connection link and check if you have a stable network connection." = "Если ваш контакт не удалил соединение или эта ссылка уже не использовалась, это может быть программная ошибка - пожалуйста, сообщите нам об этом. Чтобы установить соединение, попросите ваш контакт создать еще одну ссылку-приглашение и проверьте, стабильно ли у вас сетевое соединение.";
```

Comment:
```suggestion
"Unless your contact deleted the connection or this link was already used, it might be a bug - please report it. To connect, please ask your contact to create another connection link and check if you have a stable network connection." = "Возможно, ваш контакт удалил ссылку, или она уже была использована. Если это не так, то это может быть ошибкой - сообщите нам об этом, пожалуйста.\nЧтобы установить соединение, попросите ваш контакт создать еще одну ссылку и проверьте вашу сотовую сеть.";
```